docs/manual/cluster-admin/basic-management-operations.md

```diff
@@ -151,4 +151,117 @@ Another example is to restart the whole cluster:
 ./paictl.py service start
 ```
 
-You can use `exit` to leave the dev-box container, and use `sudo docker exec -it dev-box bash` to re-enter it if you desire so. If you don't need it any more, use `sudo docker stop dev-box` and `sudo docker rm dev-box` to delete the docker container.
\ No newline at end of file
+You can use `exit` to leave the dev-box container, and use `sudo docker exec -it dev-box bash` to re-enter it if you desire so. If you don't need it any more, use `sudo docker stop dev-box` and `sudo docker rm dev-box` to delete the docker container.
+
+## <div id="how-to-set-up-https">How To Set Up Https</div>
+
+To configure https certificate for pylon, you need to obtain a digital certificate first, and then save the digital certificate related files in the dev-box container. In the dev-box, you can find the configuration file `services-configuration.yaml`, and then you need to Enter the file path of the saved digital certificate into the `services-configuration.yaml` file. You can choose a self-signed certificate or a certificate issued by a CA. Next, we will first demonstrate the configuration process of a self-signed certificate. The configuration process for the two types of certificates is similar.
```

Comment:
` digital certificate related files` -> ` digital certificate-related files`
` In the dev-box` -> ` In the dev-box container`
`Enter the file path` -> `add the file path`
`The configuration process for the two types of certificates is similar.` -> `The configuration processes for the two types of certificates are similar.`